test/e2e/testnet/node.go

```diff
@@ -160,7 +165,7 @@ func NewNode(
 	if err != nil {
 		return nil, err
 	}
-	args := []string{"start", fmt.Sprintf("--home=%s", remoteRootDir), "--rpc.laddr=tcp://0.0.0.0:26657"}
+	args := []string{"start", fmt.Sprintf("--home=%s", remoteRootDir), fmt.Sprintf("--address=0.0.0.0:%d", listenAddressPort), fmt.Sprintf("--grpc.address=0.0.0.0:%d", grpcPort), "--rpc.laddr=tcp://0.0.0.0:26657", fmt.Sprintf("--rpc.grpc_laddr=tcp://0.0.0.0:%d", grpcListenPort)}
```

Comment:
in sdk 46, the defaults for these bound to 0.0.0.0, however they are now defaulting to binding to 127.0.0.1, which means we need to explicitly bind them to 0.0.0.0 so they are accessible to other hosts.